app/views/collections/_deposit_button.html.erb

```diff
@@ -1,5 +1,8 @@
 <%= turbo_frame_tag dom_id(collection, :deposit) do %>
-  <% if allowed_to?(:create?, WorkVersion.new(work: Work.new(collection: collection))) %>
+  <%
+    if allowed_to?(:create?, WorkVersion.new(work: Work.new(collection: collection))) &&
+       allowed_to?(:deposit?, collection.head)
```

Comment:
Do not show the deposit/reserve PURL buttons if the collection is not deposit-intoable.